modules/300-prometheus/templates/prometheus/prometheus.yaml

```diff
@@ -65,6 +65,8 @@ spec:
   additionalArgs:
     - name: scrape.timestamp-tolerance
       value: 10ms
+  scrapeConfigNamespaceSelector: {}
+  scrapeConfigSelector: {}
```

Comment:
ns `d8-monitoring` and label `additional-configs-for-prometheus=main`
same as secrets